tensorflow/core/platform/default/gpu/cupti_wrapper.h

```diff
@@ -23,7 +23,7 @@ limitations under the License.
 #if defined(WIN32)
 #include "extras/CUPTI/include/cupti.h"
 #else
-#include "cuda/extras/CUPTI/include/cupti.h"
+#include "cupti.h"
```

Comment:
This seems like it would break the existing non-Windows GPU build, wouldn't it?

Reply:
@mrry Thanks for the review. In 
https://github.com/tensorflow/tensorflow/blob/968addadfd4e4f5688eedc31f92a9066329ff6a7/third_party/gpus/cuda/BUILD.tpl#L167-L178

The `cupti_headers` actually adds `"cuda/extras/CUPTI/include/"` to the include path.

As `cupti_wrapper.h` depends on https://github.com/tensorflow/tensorflow/blob/968addadfd4e4f5688eedc31f92a9066329ff6a7/tensorflow/core/platform/default/gpu/BUILD#L18

The bazel build for GPU build should be fine I think.